HW02_B_Mimic_starter - Viola graphs.Rmd

```diff
@@ -0,0 +1,178 @@
+---
```

Comment:
So in order for the graphs you made to be viewable on github you need to just quickly go back into  RStudio and "knit" your .Rmd files that you already edited. That will create an .md file with the same name  "HW02_B_Mimic_starter - Viola graphs.md" that you should commit, push and pull to the main repo. Then when I go to look at it on here I'll be able to see your code and your graphs at the same time :)

Reply:
Ah! Thank you! I'll try fixing it!

Reply:
No problem! I think a lot of people don't know this so it's hard to comment on each other's graphs lol